template-parts/header/builder/mobile-header-row.php

```diff
@@ -5,78 +5,75 @@
 
-	if ( 'primary' === $row && $is_transparent_header_enable ) {
-		$customizer_editor_row = 'section-transparent-header';
+	if ( 'primary' === $astra_row && $astra_is_transparent_header_enable ) {
+		$astra_customizer_editor_row = 'section-transparent-header';
 	}
 
-	$row_label = ( 'primary' === $row ) ? 'main' : $row;
+	$astra_row_label = ( 'primary' === $astra_row ) ? 'main' : $astra_row;
 	?>
-	<div class="ast-<?php echo esc_attr( $row_label ); ?>-header-wrap <?php echo 'primary' === $row ? 'main-header-bar-wrap' : ''; ?>" >
-		<div class="<?php echo esc_attr( 'ast-' . $row . '-header-bar ast-' . $row . '-header ' ); ?><?php echo 'primary' === $row ? 'main-header-bar ' : ''; ?>site-<?php echo esc_attr( $row ); ?>-header-wrap site-header-focus-item ast-builder-grid-row-layout-default ast-builder-grid-row-tablet-layout-default ast-builder-grid-row-mobile-layout-default" data-section="<?php echo esc_attr( $customizer_editor_row ); ?>">
+	<div class="ast-<?php echo esc_attr( $astra_row_label ); ?>-header-wrap <?php echo 'primary' === $astra_row ? 'main-header-bar-wrap' : ''; ?>" >
+		<div class="<?php echo esc_attr( 'ast-' . $astra_row . '-header-bar ast-' . $astra_row . '-header ' ); ?><?php echo 'primary' === $astra_row ? 'main-header-bar ' : ''; ?>site-<?php echo esc_attr( $astra_row ); ?>-header-wrap site-header-focus-item ast-builder-grid-row-layout-default ast-builder-grid-row-tablet-layout-default ast-builder-grid-row-mobile-layout-default" data-section="<?php echo esc_attr( $astra_customizer_editor_row ); ?>">
 				<?php
 				if ( is_customize_preview() ) {
-					Astra_Builder_UI_Controller::render_grid_row_customizer_edit_button( 'Header', $row );
+					Astra_Builder_UI_Controller::render_grid_row_customizer_edit_button( 'Header', $astra_row );
 				}
 				/**
 				 * Astra Render before Site Content.
 				 */
-				do_action( "astra_header_{$row}_container_before" );
+				do_action( "astra_header_{$astra_row}_container_before" );
 				?>
-					<div class="ast-builder-grid-row <?php echo Astra_Builder_Helper::has_mobile_side_columns( $row, 'header', 'mobile' ) ? 'ast-builder-grid-row-has-sides' : 'ast-grid-center-col-layout-only ast-flex'; ?> <?php echo Astra_Builder_Helper::has_mobile_center_column( $row, 'header', 'mobile' ) ? 'ast-grid-center-col-layout' : 'ast-builder-grid-row-no-center'; ?>">
-						<?php if ( Astra_Builder_Helper::has_mobile_side_columns( $row, 'header', 'mobile' ) ) { ?>
-							<div class="site-header-<?php echo esc_attr( $row ); ?>-section-left site-header-section ast-flex site-header-section-left">
+					<div class="ast-builder-grid-row <?php echo Astra_Builder_Helper::has_mobile_side_columns( $astra_row ) ? 'ast-builder-grid-row-has-sides' : 'ast-grid-center-col-layout-only ast-flex'; ?> <?php echo Astra_Builder_Helper::has_mobile_center_column( $astra_row ) ? 'ast-grid-center-col-layout' : 'ast-builder-grid-row-no-center'; ?>">
```

Comment:
```suggestion
					<div class="ast-builder-grid-row <?php echo Astra_Builder_Helper::has_mobile_side_columns( $astra_row, 'header', 'mobile' ) ? 'ast-builder-grid-row-has-sides' : 'ast-grid-center-col-layout-only ast-flex'; ?> <?php echo Astra_Builder_Helper::has_mobile_center_column( $astra_row, 'header', 'mobile' ) ? 'ast-grid-center-col-layout' : 'ast-builder-grid-row-no-center'; ?>">
```
@imnavanath  I think this should be updated, please check all other same functions.

Reply:
I think no need of this - https://github.com/brainstormforce/astra/blob/next-release/inc/core/builder/class-astra-builder-helper.php#L870-L880
Only 1 parameter provided & processed here.